### ISSUE TITLE
Delete the src/config.js file

### ISSUE THREAD

**Issue by yannbertrand** [YannBertrand/YouWatch#9]
Google client credentials should be stored with configstore.

At first launch, we should ask for these credentials and the user can use default ones if they don't want to go to the Google console (an alert should be displayed to inform that the app may not work in that case).

Width, height and position config should also be stored with configstore. They would be updated each time the user changes them. It would also be great if we save both the normal mode and the float-on-top mode position/sizes.

**Comment by benavern** [comment]
I agree with most of this but the different positions of the window in normal or float-on-top. I think this has to be the same.

**Comment by yannbertrand** [comment]
We may need to add an option inside the config page then!

I'd personally prefer the solution I described in this issue as I would add videos in the playlist, launch on the float-on-top mode in the bottom right hand corner of my screen and never touch it again :smile:

**Comment by benavern** [comment]
maybe we could 
- keep the window position by default
- put a setting to always move the window on a (choosed) screen side with predefined sizes when the user  goes fullscreen.

**Comment by yannbertrand** [comment]
Asking for this when the user resize or move the window (in float-on-top mode) would be :fire:. Just having a checkbox in the config page would be fine by now :wink:
